Source/EasyNetQ/AdvancedBusExtensions.Publish.cs

```diff
@@ -22,16 +22,18 @@ public static partial class AdvancedBusExtensions
     /// If this flag is true, the server will return an unroutable message with a Return method.
     /// If this flag is false, the server silently drops the message.
     /// </param>
+    /// <param name="confirms">todo</param>
     /// <param name="message">The message to publish</param>
     /// <param name="cancellationToken">The cancellation token</param>
     public static Task PublishAsync(
         this IAdvancedBus bus,
         in Exchange exchange,
         string routingKey,
         bool? mandatory,
+        bool? confirms,
```

Comment:
@Pliner is this name good enough? Or can you suggest something better?

Reply:
renamed to `publisherConfirms` the same as ConnectionConfiguration property name

Reply:
Ok, it makes sense.